src/app/app.server.ts

```diff
@@ -0,0 +1,15 @@
+import dotenv from 'dotenv';
+import express, { Express } from 'express';
+
+dotenv.config({
+  path: '.env'
+});
+
+const app: Express = express();
+
+export function initServer() {
+  const port: string = process.env.APP_PORT || '3000';
```

Comment:
You should load the port in the index file, and then send the port as a parameter in the function `initServer(port)`